Index: target/classes/data/Sup.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>apple*qua tao\r\ncuong03*----\r\ncuong04*----\r\ncuong05*----\r\ncuong06*----\r\ncuong07*----\r\ncuong08*----\r\ncuong10*----\r\ncuong11*----\r\ncuong12*----\r\nhouse*nhà\r\nplay*chơi\r\n
===================================================================
diff --git a/target/classes/data/Sup.txt b/target/classes/data/Sup.txt
--- a/target/classes/data/Sup.txt	
+++ b/target/classes/data/Sup.txt	
@@ -1,3 +1,4 @@
+ahihi*cười
 apple*qua tao
 cuong03*----
 cuong04*----
